### DISCUSSION TITLE
Can watchtower watch itself?

### DISCUSSION THREAD

**Discussion by svh1985** [containrrr/watchtower#611] [Q&A] [answered]
Can I apply the label to watchtower so it updates itself?
```
    labels: 
      com.centurylinklabs.watchtower.enable: "true"
```

**Comment by victorcmoura** [comment, marked as answer]
Hi, @svh1985. 

> Can I apply the label to watchtower so it updates itself?

Yes, you can. 

As far as I could understand from the source code, Watchtower should change its own container name, run the updated copy (so, at least temporarily, there will be two instances running at the same time), and stop the oldest.

**Reply by victorcmoura** [reply]
@simskij has confirmed this feature in https://github.com/containrrr/watchtower/discussions/565
